tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -329,7 +329,7 @@ func NewLabelSchedulerCommand() *cobra.Command {
 func NewGrantHotRegionSchedulerCommand() *cobra.Command {
 	c := &cobra.Command{
 		Use:   "grant-hot-region-scheduler <store_leader_id> <store_leader_id,store_peer_id_1,store_peer_id_2>",
-		Short: "add a scheduler to grant hot region to fixed stores",
+		Short: "add a scheduler to grant hot region to fixed stores. Note: balance-hot-region-scheduler must be paused.",
```

Comment:
Could we do some actual checks to prevent these two schedulers from working at the same time?

Reply:
Good idea.